TRANSCRIPTION.md

```diff
@@ -75,27 +75,21 @@
 | ð | 3 | ✓ | ✓ |
 | õ | 3 | ✓ | ✓ |
 | ɾʰ | 3 | ✓ | ✓ |
-| aʰ | 2 | ? | ? |
 | aˀ | 2 | ✓ | ✓ |
 | hʲ | 2 | ✓ | ✓ |
 | z | 2 | ✓ | ✓ |
 | ãː | 2 | ✓ | ✓ |
 | ə̃ | 2 | ✓ | ✓ |
-| iʰ | 1 | ? | ? |
 | i̥ | 1 | ✓ | ✓ |
 | jʰ | 1 | ✓ | ✓ |
-| ka | 1 | ? | ? |
 | kw | 1 | ✓ | ✓ |
```

Comment:
This is ambiguous in CLTS. Either it should be two sounds, or k_w. But kw is interpreted as a double articulation clusuter. So it should not be used in the profile.

Reply:
There is no profile, it is using the segmentation provided by the authors (Chacon, I assume). I'll check for cognates to see what it is supposed to be.

Reply:
Okay, another case for segments, then, right?